report_pre_thesis/src/sections/methodology.tex

```diff
@@ -1,8 +1,246 @@
 \section{Methodology}\label{sec:methodology}
-Method / Our Contribution (Detailed main part of the story, or how the analysis was done)
+As part of our goal to evaluate the performance of each of the components in the pipeline, the first step was to create a replica of the original pipeline described by \citet{cleggRecalibrationMarsScience2017}
+Since we did not have access to the original source code implementing the pipeline, we have replicated it at accurately as we could based on the available information.
+However, not all details were included in \citet{cleggRecalibrationMarsScience2017}, so we have had to make some assumptions, which we detail here.
+In addition, some aspects of the pipeline rely on qualitative assessments made by the original authors --- something we cannot do because we are not domain experts.
+Consequently, our pipeline is not identical to the original, but we have strived to make it as close as possible.
 
-\textit{Recreate the current model. Reproduce the results of the original papers. Investigate the limitations of the model and propose improvements.}
+This section is dedicated to describing the methodology of our pipeline, how it differs from the original, and which design choices we have made and why.
+Furthermore, we delve into the experiments we have conducted to evaluate the performance of our pipeline such that we can identify the components that contribute the most to the overall error.
 
+Section \ref{sec:shared_preprocessing} outlines the preprocessing steps common to both the PLS1-SM and ICA phases of the pipeline.
+In Section \ref{sec:methodology_pls1}, we focus on the specifics of the PLS1-SM phase, detailing its associated data preprocessing procedures.
+Section \ref{sec:methodology_ica} then details the ICA phase, including its unique data preprocessing steps.
+Lastly, Section \ref{sec:methodology_moc} presents a description of the MOC phase of the pipeline.
+
+\subsection{Shared Preprocessing}\label{sec:shared_preprocessing}
+Similar to the original pipeline, we have used the samples from the ChemCam calibration dataset.
```

Comment:
The suggestion is fine but we usually use \citet instead of \citeauthor so we get both the author name and citation in this section.

The reason we describe the data at this point is that our supervisors asked us to explicitly describe the data format at each stage, including pre-processing. Describing the entire pre-processing in detail was done to adhere to this request from them.